docker-compose.dev.yml

```diff
@@ -81,6 +83,9 @@ services:
     container_name: lago_api
     restart: unless-stopped
     command: ["./scripts/start.dev.sh"]
+    depends_on:
+      - db
+      - clickhouse
```

Comment:
Clickhouse is an optional dependency, we do not want to make it mandatory to start the application

Reply:
Makes sense